File: beginner/__1000__/1041/1041.ts
const fs = require("fs")
const input = fs.readFileSync('/dev/stdin', "utf-8")
const [a, b] = input.split(' ').map(Number)

if(a === 0.0 && b === 0.0){console.log("Origem")}
else if (a == 0.0){console.log("Eixo Y");}
else if (b == 0.0){console.log("Eixo X");}
else if (b > 0.0 && a > 0.0){console.log("Q1");}
else if (b > 0.0 && a < 0.0){console.log("Q2");}
else if (b < 0.0 && a < 0.0){console.log("Q3");}
else if (b < 0.0 && a > 0.0){console.log("Q4");}

File: beginner/__1000__/1041/1041.py
a, b = [float(n) for n in input().split()]

if a == 0 and b == 0:
    print("Origem")
elif  a == 0:
    print("Eixo Y")
elif  b == 0:
    print("Eixo X")
elif  b > 0 and a > 0:
    print("Q1")
elif  b > 0 and a < 0:
    print("Q2")
elif  b < 0 and a < 0:
    print("Q3")
elif  b < 0 and a > 0:
    print("Q4")